### PR TITLE
Current Code in the Alpha version of app

### PR DESCRIPTION
**Pull request by kexinF**

### CODE REVIEW

File: lib/zoomapi.ts [hunk 1]
```diff
@@ -22,14 +22,16 @@ class ZoomApiImpl implements ZoomApiWrapper {
...
+    // console.log(`Zoom SDK config response: ${configResponse}`);
     return zoomSdk.setVirtualForeground({imageData});
   }
 
   async removeVirtualForeground(): Promise<GeneralMessageResponse> {
-    const configResponse = await this.initialize();
-    console.log(`Zoom SDK config response: ${configResponse}`);
+    // const configResponse = await this.initialize();
+    await this.initialize();
+    // console.log(`Zoom SDK config response: ${configResponse}`);
```
Comment (blickly) [inline lib/zoomapi.ts:34]: Can we remove the commented out code?

File: lib/drawNametag.ts [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    context.font = 'bold 80px Arial'; // Larger font size
+    context.fillText(out.substring(0, 3), 70, 150); // Draw the first character
+    context.font = 'bold 30px Arial';
+    context.fillText(out.substring(3), 160, 130);
+
+  }
+
+  const newImageData = context.getImageData(0, 0, canvas.width, canvas.height);
+
+  // console.log(newImageData)
```
Comment (blickly) [inline lib/drawNametag.ts:69]: Can we remove the commented out code?

File: app/main/Affirmation.tsx [hunk 3]
```diff
@@ -1,36 +1,43 @@
+// Under Review
+
 "use client";
 import React, { useState, useEffect } from 'react';
-import Header from '../header';
-import Footer from '../footer';
 import Modal from 'react-modal';
-import { affirmations } from '../state';
+// import { useCustomState } from './state';
```
Comment (blickly) [inline app/main/Affirmation.tsx:6]: Can we remove the commented out code?

File: app/main/Affirmation.tsx [hunk 4]
```diff
@@ -1,36 +1,43 @@
...
+}
+
+
+function Affirmation({
+  allAffirmations,
+  setCurrentAffirmation,
+  setAllAffirmations,
+}: AffirmationProps) {
+
+  // const { state, setCurrentAffirmation, setAllAffirmations  } = useCustomState();
```
Comment (blickly) [inline app/main/Affirmation.tsx:27]: Can we remove the commented out code?

File: app/main/Affirmation.tsx [hunk 5]
```diff
@@ -1,36 +1,43 @@
...
-
-  // Initialize state with saved data if it exists, otherwise use the initial data.
-  const [buttons, setButtons] = useState<Button[]>(() => {
-    const stringifiedAffirmations = affirmations.getAffirmationsAsString();
-    return stringifiedAffirmations ? JSON.parse(stringifiedAffirmations) : initialButtons;
-  });
+
+  const [buttons, setButtons] = useState(allAffirmations)
+
+  // console.log('applyed', state.selectedAffirmation)
```
Comment (blickly) [inline app/main/Affirmation.tsx:36]: Can we remove the commented out code?

File: app/main/Affirmation.tsx [hunk 6]
```diff
@@ -44,11 +51,17 @@ function Home() {
 
   const handleEdit = () => {
     setButtons((prevButtons) =>
-      prevButtons.map((button) =>
-        button.id === currentEditId ? { ...button, text: editText } : button
-      )
+      prevButtons.map((button) => {
+        // console.log('Original text:', button.text);
```
Comment (blickly) [inline app/main/Affirmation.tsx:55]: Can we remove the commented out code?

File: app/main/Affirmation.tsx [hunk 7]
```diff
@@ -44,11 +51,17 @@ function Home() {
...
+
+        if (button.id === currentEditId) {
+          // console.log('Edited text:', editText);
```
Comment (blickly) [inline app/main/Affirmation.tsx:58]: Can we remove the commented out code?

File: app/main/NameTag.tsx [hunk 8]
```diff
@@ -0,0 +1,152 @@
...
+
+  useEffect(() => {
+    debouncedEffect();
+    return debouncedEffect.cancel;
+  }, [showNametag, inputValues, selectedWaveHand, waveHands]);
+
+  const handleInputChange = (index: number, value: string) => {
+    const newInputValues = [...inputValues];
+    newInputValues[index] = value;
+    // console.log(value)
```
Comment (blickly) [inline app/main/NameTag.tsx:66]: Can we remove the commented out code?

File: app/main/Tab.tsx [hunk 9]
```diff
@@ -0,0 +1,110 @@
...
+"use client";
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+interface TabProps {
+  activeTab: string;
+  'page-label': string;
+  onClick: (label: string) => void;
+  // children: React.ReactNode;
```
Comment (blickly) [inline app/main/Tab.tsx:12]: Can we remove the commented out code?

File: app/main/page.tsx [hunk 10]
```diff
@@ -0,0 +1,106 @@
+// Under Review
+
+"use client";
+
+import React from 'react';
+import Tabs from "./Tabs";
+// import Tab from "./Tab";
```
Comment (blickly) [inline app/main/page.tsx:7]: Can we remove the commented out code?

File: app/main/page.tsx [hunk 11]
```diff
@@ -0,0 +1,106 @@
...
+const apiList: Apis[] = [
+  "setVirtualForeground",
+  "removeVirtualForeground",
+];
+
+function App() {
+
+  const { state, 
+  setSelectedWaveHand,
+  // setHandChoicesAsString,
```
Comment (blickly) [inline app/main/page.tsx:27]: Can we remove the commented out code?

File: app/main/state.ts [hunk 12]
```diff
@@ -0,0 +1,110 @@
...
+
+  const setSelectedWaveHand = (newSelectedWaveHand: number) => {
+    setState((prevState) => ({
+      ...prevState,
+      selectedWaveHand: prevState.selectedWaveHand === newSelectedWaveHand ? null : newSelectedWaveHand,
+    }));
+  };
+
+
+  // const setHandChoicesAsString = (hands: string[]) => {
```
Comment (blickly) [inline app/main/state.ts:53]: Can we remove the commented out code?

File: app/main/state.ts [hunk 13]
```diff
@@ -0,0 +1,110 @@
...
+      if (JSON.stringify(prevState.currentNameTag) !== JSON.stringify(newNametag)) {
+        return {
+          ...prevState,
+          currentNameTag: newNametag,
+        };
+      }
+      return prevState;
+    });
+
+    // console.log(newNametag);
```
Comment (blickly) [inline app/main/state.ts:86]: Can we remove the commented out code?

**Review by blickly — CHANGES_REQUESTED** [review]
Looks good! Before committing let's first:

* Fix the CI (looks like just a merge issue with two `const` variables with the same name)
* Remove commented out code

Other than that, I think we can address much more in smaller reviews of smaller batches of files.

File: app/main/state.ts (hunk 13 above)
Comment (kexinF) [reply app/main/state.ts:79]: @blickly I have removed all these comments. I also cleared the build error. Let me know if I happen to miss any comments!

**Review by blickly — APPROVED** [review]
Sounds good. Let's merge this now and review the code in more detail in smaller pieces

**Pull request merged**